src/profile-logic/marker-data.js

```diff
@@ -321,6 +321,16 @@ export function isNetworkMarker(marker: TracingMarker): boolean {
   return !!(marker.data && marker.data.type === 'Network');
 }
 
+export function filterForNetworkChart(markers: TracingMarker[]) {
+  return markers.filter(marker => isNetworkMarker(marker));
+}
+
+export function filterForMarkerChart(markers: TracingMarker[]) {
+  return markers.filter(marker => !isNetworkMarker(marker));
+}
+
+// TODO: add function to merge start and end markers
```

Comment:
I can remove the comment, but as this will happen pretty much next (thanks to the patch Randell provided - https://bugzilla.mozilla.org/show_bug.cgi?id=1490963), I'd appreciate the pointer.